### PR TITLE
Merge GATKProtectedVariantContextUtils methods into other classes

### LINKED ISSUE #2776: consolidate classes from gatk-protected that mirror things from public
There are a number of classes in protected that mirror classes in gatk.  

ex: `GATKProtectedMathUtils`, `GATKProtectedVariantContextUtils`

We should combine these with the class in public so we don't have arbitrary differences between them.

### PR DESCRIPTION
**Pull request by davidbenjamin**
Closes #2776.  @lbergelson There is nothing of substance here, just moving things around

### CODE REVIEW

File: src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/RMSMappingQuality.java [hunk 1]
```diff
@@ -25,8 +25,8 @@
 
 import java.util.*;
 
-import static org.broadinstitute.hellbender.utils.GATKProtectedVariantContextUtils.getAttributeAsLong;
-import static org.broadinstitute.hellbender.utils.GATKProtectedVariantContextUtils.getAttributeAsLongList;
+import static org.broadinstitute.hellbender.utils.variant.VariantContextGetters.getAttributeAsLong;
```
Comment (lbergelson) [inline src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/RMSMappingQuality.java]: We typically don't use static imports at all.

**Comment by davidbenjamin** [comment]
> Why move the getAsX methods to a new class instead of just into GATKVariantContextUtils? It seems hard to discover there?

`GATKVariantContextUtils` was already bloated, and after this PR it's closing in on 2000 lines.  I thought I would extract some methods that follow a theme.

> I take it that unused methods were deleted?

Just a couple of them, but yes.

File: src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/RMSMappingQuality.java (hunk 1 above)
Comment (davidbenjamin) [reply src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/RMSMappingQuality.java]: I blame IntelliJ.

Comment (davidbenjamin) [reply src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/RMSMappingQuality.java]: fixed it

**Pull request merged**